packages/discord.js/src/structures/UncachedGuildMember.js

```diff
@@ -0,0 +1,328 @@
+'use strict';
+
+const Base = require('./Base');
+const TextBasedChannel = require('./interfaces/TextBasedChannel');
+const { GuildMemberFlagsBitField } = require('../util/GuildMemberFlagsBitField');
+const PermissionsBitField = require('../util/PermissionsBitField');
+
+/**
+ * Represents a member of a guild on Discord. Used in interactions from guilds that aren't cached.
+ * Backwards compatible with {@link APIGuildMember}.
+ * @implements {TextBasedChannel}
+ * @extends {Base}
+ */
+class UncachedGuildMember extends Base {
+  constructor(client, data, guildId) {
+    super(client);
+
+    /**
+     * The ID of the guild that this member is part of
+     * @type {string}
+     */
+    this.guildId = guildId;
+
+    /**
+     * The user that this guild member instance represents
+     * @type {User}
+     */
+    this.user = this.client.users._add(data.user, true);
```

Comment:
Isn’t this a breaking change? `APIInteractionGuildMember#user` is `APIUser` and thus has snakecase properties. Now it doesn’t anymore.

Reply:
Yep, that is a breaking change indeed... @advaith1 can you solve that too?